amundsen_application/static/webpack.dev.ts

```diff
@@ -1,4 +1,27 @@
+import * as path from 'path';
 import merge from 'webpack-merge';
-import commonConfig from './webpack.common'
+import commonConfig, {htmlWebpackPluginConfig} from './webpack.common';
+import MiniCssExtractPlugin from 'mini-css-extract-plugin';
 
-export default merge(commonConfig, {});
+export default merge(commonConfig, {
+  devServer: {
+      contentBase: path.join(__dirname, '../build/lib/amundsen_application/static/dist'),
+      publicPath: '/static/dist/',
+      watchContentBase: true,
+      port: 60480,
+      proxy: {
```

Comment:
What behavior does this contribute to, and what exists at the specified target?

Reply:
The proxy says that if you get a request for anything other than these urls, send it back to the original application port. This port is incorrect as is, however, and I need to switch it back to the port you use.